website/source/api/connect/intentions.html.md

```diff
@@ -165,11 +165,16 @@ The table below shows this endpoint's support for
 <sup>1</sup> Intention ACL rules are specified as part of a `service` rule.
 See [Intention Management Permissions](/docs/connect/intentions.html#intention-management-permissions) for more details.
 
+### Parameters
+
+- `filter` `(string: "")` - Specifies the expression used to filter the
+  queries results prior to returning the data.
+
 ### Sample Request
 
 ```text
 $ curl \
-    http://127.0.0.1:8500/v1/connect/intentions
+    'http://127.0.0.1:8500/v1/connect/intentions?SourceName==web'
```

Comment:
I was just scanning over this and noticed this might be the wrong query param, should it be `?filter` not `?SourceName`?

Reply:
Yep, apparently I missed the 'filter=' part there.